src/DssSpell.t.sol

```diff
@@ -2211,66 +2215,50 @@ contract DssSpellTest is DSTest, DSMath {
-        assertEq(flash.vat(), address(vat));
-        assertEq(flash.daiJoin(), address(daiJoin));
-        assertEq(flash.dai(), address(dai));
-        assertEq(flash.vow(), address(vow));
-        assertEq(flash.max(), 500 * MILLION * WAD);
-        assertEq(flash.toll(), 5 * WAD / 10000);
-        assertEq(flash.maxFlashLoan(address(dai)), 500 * MILLION * WAD);
-        assertEq(flash.flashFee(address(dai), 1 * MILLION * WAD), 500 * WAD); // 500 DAI fee on a 1M loan
-        flash.flashLoan(address(this), address(dai), 1 * MILLION * WAD, "");
-        flash.vatDaiFlashLoan(address(this), 1 * MILLION * RAD, "");
-        assertEq(vat.sin(address(flash)), 0);
-        assertEq(vat.dai(address(flash)), 1000 * RAD);
-        flash.accrue();
-        assertEq(vat.dai(address(flash)), 0);
-        assertEq(vat.dai(address(vow)), vowDai + 1000 * RAD);
-    }
-
-    function onFlashLoan(
-        address initiator,
-        address token,
-        uint256 amount,
-        uint256 fee,
-        bytes calldata
-    ) external returns (bytes32) {
-        assertEq(initiator, address(this));   
-        assertEq(token, address(dai));
-        assertEq(amount, 1 * MILLION * WAD);
-        assertEq(fee, 500 * WAD);
-
-        dai.approve(msg.sender, 1_000_500 * WAD);
-
-        return keccak256("ERC3156FlashBorrower.onFlashLoan");
-    }
+        assertEq(vat.can(address(pauseProxy), address(daiJoin)), 1);
+
+        assertEq(
+            vat.sin(address(vow)) - prevSin,
+            ( amountGro
+            + amountMkt
+            + amountGov
+            + amountRwf
+            + amountRisk
+            + amountPe
+            + amountOra
+            + amountOraEr
+            ) * RAD
+        );
+        assertEq(vat.sin(address(vow)) - prevSin, amountTotal * RAD);
+        assertEq(dai.balanceOf(GRO_MULTISIG)    - prevDaiGro, amountGro * WAD);
+        assertEq(dai.balanceOf(MKT_MULTISIG)    - prevDaiMkt, amountMkt * WAD);
+        assertEq(dai.balanceOf(GOV_MULTISIG)    - prevDaiGov, amountGov * WAD);
+        assertEq(dai.balanceOf(RWF_MULTISIG)    - prevDaiRwf, amountRwf * WAD);
+        assertEq(dai.balanceOf(RISK_MULTISIG)   - prevDaiRisk, amountRisk * WAD);
+        assertEq(dai.balanceOf(PE_MULTISIG)     - prevDaiPe, amountPe * WAD);
+        assertEq(dai.balanceOf(ORA_MULTISIG)    - prevDaiOra, amountOra * WAD);
+        assertEq(dai.balanceOf(ORA_ER_MULTISIG) - prevDaiOraEr, amountOraEr * WAD);
 
-    function onVatDaiFlashLoan(
-        address initiator,
-        uint256 amount,
-        uint256 fee,
-        bytes calldata
-    ) external returns (bytes32) {
-        assertEq(initiator, address(this));   
-        assertEq(amount, 1 * MILLION * RAD);
-        assertEq(fee, 500 * RAD);
-
-        dai.approve(address(daiJoin), 500 * WAD);
-        daiJoin.join(address(this), 500 * WAD);
-        vat.move(address(this), msg.sender, 1_000_500 * RAD);
-
-        return keccak256("VatDaiFlashBorrower.onVatDaiFlashLoan");
     }
```

Comment:
There's a test we should probably add: that the PauseProxy can claw back any funds sent to the Oracle Operational multisig. Have to call `executeAllowanceTransfer` on https://etherscan.io/address/0xcfbfac74c26f8647cbdb8c5caf80bb5b32e43134 .

Although there's a separate issue here that the multisig owners can revoke the allowance at any time, so this "clawback ability" may just be theater:
https://forum.makerdao.com/t/mip40c3-sp15-modify-oracles-core-unit-budget/8047/11

Reply:
Added test.